scaffolding-chef-inspec/tests/user-linux-waiver/tests/test.bats

```diff
@@ -0,0 +1,15 @@
+@test "API: scaffold_cacerts matches run hook core/cacerts" {
+  result="$(grep '^export SSL_CERT_FILE.*' /hab/svc/${TEST_PKG_NAME}/hooks/run | cut -d/ -f4-5)"
+  [ "${result}" = "core/cacerts" ]
+}
+
+@test "API: scaffold_cacerts matches run hook core/cacerts" {
+  result="$(grep '^export SSL_CERT_DIR.*' /hab/svc/${TEST_PKG_NAME}/hooks/run | cut -d/ -f4-5)"
+  [ "${result}" = "core/cacerts" ]
+}
+
+teardown(){
+  if [ "${BATS_TEST_NUMBER}" -eq ${#BATS_TEST_NAMES[@]} ]; then
+    hab svc unload "${TEST_PKG_IDENT}" || true
+  fi
```

Comment:
Is there a way to test the waiver.yml file here?

Reply:
I think so I'll add a test for that. One thing I've been struggling with is I'd like the CI to succeed or fail based on the InSpec run but I'm not sure if there is a way to read the supervisor log for content. I'll keep playing with it and see if I can figure it out.

Reply:
You could have the report send to another running service on the server. Then pull the report out and test the results. We are doing this in automate to test notifications. 

Below is the service Automate is using to receive the report. 
https://github.com/chef/automate/blob/609bfde077576aea1ec94896dfc5bb113456690a/integration/helpers/requestbin/requestbin.go#L25

If you wanted to go down that path, this project might be a good place to start. 
https://github.com/lancewf/notifications-service

This project is set up to be a lightweight Automate. Where InSpec reports will be sent to it and it will send out notifications via email, slack, or phone if the report shows failure.

Reply:
Awesome Thank you!!! I'll take a look.